ui/src/app/components/runs/runs.component.html

```diff
@@ -60,7 +60,13 @@
   </clr-dg-row>
 
   <clr-dg-detail *clrIfDetail="let detail">
-    <clr-dg-detail-header>{{detail.workflowName}} ({{detail.projectName}}) - {{detail.started | date:'medium'}} - {{detail.finished | date:'medium'}}</clr-dg-detail-header>
+    <clr-dg-detail-header>
```

Comment:
Hmm.. It looks a bit "in your face", tbh. But I don't have a better idea right now

Reply:
Yes, bit. We can take a look on this later.

Reply:
Ok